web-console/src/views/home-view.tsx

```diff
@@ -332,6 +367,18 @@ GROUP BY 1`);
         error: state.datasourceCountError
       })}
 
+      {this.renderCard({
+        href: '#tasks',
+        icon: IconNames.LIST_COLUMNS,
+        title: 'Supervisors',
+        loading: state.supervisorCountLoading,
+        content: <>
+          <p>{pluralIfNeeded(state.runningSupervisorCount, 'running supervisor')}</p>
+          <p>{pluralIfNeeded(state.suspendedSupervisorCount, 'suspended supervisor')}</p>
+          </>,
```

Comment:
alignment. Also if there are no supervisors it I think it should just say "0 supervisors"